### ISSUE TITLE
Disabled pager label has insufficient contrast

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/knowledge-commons-profiles#581]
Area of Site: Profiles
URL: https://profile.hcommons-test.org/members/

**Description**
Disabled pager labels in the pagination element at the bottom of the Members page has insufficient color contrast. 

<img width="866" height="232" alt="Screenshot of a disabled pager label with insufficient contrast" src="https://github.com/user-attachments/assets/f8557a57-7631-45ae-8011-dc79c4f0380c" />

**Solution**
Change the styles of `.pager .disabled` on line 1352 of `/static/css/profile.css` like so:

```
opacity: 0.595;
box-shadow: none;
```

<img width="864" height="140" alt="Screenshot of the fixed disabled pagination label with the suggested styles applied" src="https://github.com/user-attachments/assets/21f8ac67-6f39-4d59-90b8-51e5c30e76b3" />
